source/release-notes-v4.2.17.rst

```diff
@@ -0,0 +1,22 @@
+.. _DISTMONGO-4.2.17:
+
+================================================================================
+*Percona Distribution for MongoDB* 4.2.17
+================================================================================
+
+:Date: October 5, 2021
+:Installation: `Installing Percona Distribution for MongoDB <https://www.percona.com/doc/percona-server-for-mongodb/4.2/install/index.html>`_
+
+|pdmdb| is a collection of solutions to run and operate your
+|mongodb| efficiently with the data being consistently backed up.  
+
+|pdmdb| includes the following components:
+
+* |PSMDB| is a fully compatible open source, drop-in replacement
+  for MongoDB.
+* |pbm| is a distributed, low-impact solution for achieving
+  consistent backups of |mongodb| sharded clusters and replica sets.
+
+This release of |pdmdb| is based on `Percona Server for MongoDB 4.2.17-17 <https://www.percona.com/doc/percona-server-for-mongodb/4.2/release_notes/4.2.17-17.html>`_ and `Percona Backup for MongoDB 1.6.0 <https://www.percona.com/doc/percona-backup-mongodb/release-notes/1.6.0.html>`_. 		    

```

Comment:
As per our standard we want to have Release Highlights section in all Release notes.
1. Can we add Release highlights section
2. Add some marketing touch to this sentence
3. Move it to release highlights ?